### PR TITLE
(maint) Release 0.23.0

### PR DESCRIPTION
**Pull request by jpogran**

### CODE REVIEW

**Comment by glennsarti** [comment]
Editor Services release is done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+## [0.23.0] - 2020-01-31
+
+### Added
+
+- ([GH-213](https://github.com/lingua-pupuli/puppet-editor-services/issues/213)) [puppet-editor-services-0.24.0](https://github.com/lingua-pupuli/puppet-editor-services/releases/tag/0.24.0) Gather facts using sidecar
```
Comment (glennsarti) [inline CHANGELOG.md]: Can probably squish this into a single entry.  Also this is a private implementation detail so no user impact (I hope! :-) )

Comment (jpogran) [reply CHANGELOG.md]: Merged into one entry

**Comment by jpogran** [comment]
Updated release branch and re targeted this to release

**Pull request merged**